src/cmd-oscontainer

```diff
@@ -134,6 +133,6 @@ if args.action == 'extract':
                         tls_verify=not args.disable_tls_verify)
 elif args.action == 'build':
     oscontainer_build(containers_storage, args.src, args.rev, args.name,
-                      base_image=getattr(args, 'from'),
+                      getattr(args, 'from'),
```

Comment:
Minor: we can just do `args.from` like the others now that there's a default.

Reply:
The reason for the `getattr` is that `from` is a keyword in Python.

Reply:
Ahh gotcha. IIRC there's an argparse option to map it to a different name, but yeah that's fine.